tracing-error/src/lib.rs

```diff
@@ -172,6 +175,7 @@
 //! [subscriber layer]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html
 //! [`tracing`]: https://docs.rs/tracing
 //! [`std::error::Error`]: https://doc.rust-lang.org/stable/std/error/trait.Error.html
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
remember that this needs to be added to cargo.toml as well:
https://github.com/tokio-rs/tracing/blob/86d6c1b954a95d4185e8ab8e427836a1f1820934/tracing/Cargo.toml#L76-L78

Reply:
Already there at the bottom